packages/main/src/plugin/updater.ts

```diff
@@ -163,8 +170,10 @@ export class Updater {
         buttons: buttons,
         cancelId: 1,
       });
-      if (result.response === 2) {
+      if (result.response === 3) {
         this.updateConfigurationValue('never');
+      } else if (result.response === 1) {
+        console.log('await this.openReleaseNotes(updateVersion)');
```

Comment:
?

Reply:
@axel7083 the `console.log` lines I added are necessary for this PR, but they don't work until https://github.com/containers/podman-desktop/pull/8753 is merged. Commenting them gives an error with the linter, so I just put them in `console.log`. rn this PR is not functional until https://github.com/containers/podman-desktop/pull/8753 is merged